new-site/src/components/Playground.js

```diff
@@ -9,6 +9,58 @@ import theme from 'prism-react-renderer/themes/github';
 import './Playground.scss';
 import LiveErrorCore from './LiveErrorCore';
 
+const sanitizeConfig = {
+  allowedTags: false,
+  disallowedTagsMode: 'discard',
+  allowedAttributes: {
+    '*': [
+      'aria-describedby',
+      'aria-hidden',
+      'aria-label',
+      'aria-labelledby',
+      'aria-level',
+      'class',
+      'checked',
+      'd',
```

Comment:
The pagination component has data-next attribute, we probably need to allow it?

Reply:
- [ ] add missing attribute